packages/bpk-stylesheets/base.js

```diff
@@ -63,23 +60,17 @@
 /******/ 	__webpack_require__.p = "";
 /******/
 /******/ 	// Load entry module and return exports
-/******/ 	return __webpack_require__(__webpack_require__.s = 1);
+/******/ 	return __webpack_require__(__webpack_require__.s = 0);
```

Comment:
I couldn't find in the links you posted where this change comes from - what does it do?

Reply:
extract-text-webpack-plugin added sorting https://github.com/webpack-contrib/extract-text-webpack-plugin/blob/master/CHANGELOG.md#300-rc1-2017-07-07